seacrowd/sea_datasets/thai_databricks_dolly/thai_databricks_dolly.py

```diff
@@ -0,0 +1,93 @@
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+import pandas as pd
+from datasets.download.download_manager import DownloadManager
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+# No paper citation found.
+_CITATION = ""
+
+_LOCAL = False
+_LANGUAGES = ["tha"]
+_DATASETNAME = "thai_databricks_dolly"
+_DESCRIPTION = """\
+This is a Thai-instructed dataset translated from databricks-dolly-15k using
+Google Cloud Translation. databricks-dolly-15k is an open-source dataset of
+instruction-following records generated by thousands of Databricks employees in
+several behavioral categories outlined in the InstructGPT paper, including
+brainstorming, classification, closed QA, generation, information extraction,
+open QA, and summarization.
+"""
+
+_HOMEPAGE = "https://huggingface.co/datasets/Thaweewat/databricks-dolly-15k-th"
+_LICENSE = Licenses.CC_BY_SA_3_0.value
+_URL = "https://huggingface.co/datasets/Thaweewat/databricks-dolly-15k-th/resolve/main/databricks-dolly-15k-th.parquet"
+_SUPPORTED_TASKS = [Tasks.INSTRUCTION_TUNING]
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "1.0.0"
+
+
+class ThaiDatabricksDollyDataset(datasets.GeneratorBasedBuilder):
+    """Thai Databricks Dolly Dataset"""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    SEACROWD_SCHEMA_NAME = "t2t"
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(name=f"{_DATASETNAME}_source", version=SOURCE_VERSION, description=f"{_DATASETNAME} source schema", schema="source", subset_id=_DATASETNAME),
```

Comment:
this is similar to `tgl_profanity`, pls sync the format w/ L45-L51

Reply:
Oh that's interesting. I'll clear some cache in my local env so that my formatting is consistent. you meant the formatting, no? Updated in https://github.com/SEACrowd/seacrowd-datahub/pull/242/commits/e453c02c7d4dafc30a57a15ae79d1d0da51cf8ac

Reply:
yep the code formatting. shd be okay per your latest commit